app/src/main/java/org/greenstand/android/TreeTracker/view/TreeTrackerTextButton.kt

```diff
@@ -36,7 +36,7 @@ import org.greenstand.android.TreeTracker.activities.LocalNavHostController
 import org.greenstand.android.TreeTracker.models.NavRoute
 
 @Composable
-fun TextButton(
+fun TreeTrackerTextButton(
```

Comment:
Why the rename?

Reply:
For a more conventional naming since it's custom for our TreeTracker app. I can definitely rename it however.

Reply:
I didn't noticed a pre-made TextButton, so let's try to keep TreeTracker out of names. I have done that with our theme and activity, but that was due to simple default names being used already. In fact, I'd like those names improved later in another PR.